rakelib/docs.rake

```diff
@@ -200,6 +204,14 @@ begin
       inventory   = Bolt::Config::INVENTORY_OPTIONS.dup
       @transports = Bolt::Config::TRANSPORT_CONFIG.keys
 
+      # Move sub-options for 'log' option up one level, as they're nested under
+      # 'console' and filepath
+      @opts['log'][:properties] = @opts['log'][:additionalProperties][:properties]
```

Comment:
I don't see the difference in generated outputs for this? I generated the doc on `main` and on this branch, and the results were the same...

Reply:
Run `rake docs:defaults_reference`. The template pulls from the `:properties` key, without this change running that task on its own will show `console` instead of `level` and `append` in the docs.

This change isn't needed when running `rake docs:all` since `docs:config_reference` modifies the options hash first.

Reply:
Ah I see. Ok.